packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -571,7 +577,19 @@ function makeSimulator(eventType) {
         'a component instance. Pass the DOM node you wish to simulate the event on instead.',
     );
 
-    const dispatchConfig = eventNameDispatchConfigs[eventType];
+    // Reconstruct more or less what the original event system produced.
+    // We could remove this indirection here but we also don't plan to invest in Simulate anyway.
+    const dispatchConfig = {};
```

Comment:
While a real dispatch config would have more fields, they're not used by TestUtils.